lib/internal/dtrace.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const config = internalBinding('config');
+
+const {
```

Comment:
Considering these are only used once each in a few files, maybe we can put these in `lib/internal/net.js` and `lib/internal/http.js` instead of putting them in a new module?

Reply:
I had that originally but opted for this as it eliminates some code duplication